manifests/nodejs.yml

```
@@ -298,7 +298,9 @@ tests/:
       Test_TracerSCITagging: v3.21.0
   remote_config/:
     test_remote_configuration.py:
-      Test_RemoteConfigurationExtraServices: missing_feature
+      Test_RemoteConfigurationExtraServices:
+        '*': missing_feature
+        express4: v4.17.0
```

This feature is independent from the framework no ?
```suggestion
      Test_RemoteConfigurationExtraServices: v4.17.0
```

---

So, do I need to add the /createextraservice endpoint in the typescript app?

---

In every weblog yes why not ?